src/features/web-compat.js

```diff
@@ -360,16 +358,17 @@ export class WebCompat extends ContentFeature {
             if ('credentials' in navigator && 'get' in navigator.credentials) {
                 return
             }
-            // TODO: change the property descriptor shape to match the original
             const value = {
                 get () {
                     return Promise.reject(new Error())
                 }
             }
+            // TODO: original property is an accessor descriptor
```

Comment:
What's the expected change here? Can you make it etc? Or file a follow up?

Reply:
Original descriptor is an accessor with a getter and no setter. If we go all the way, we should also shim the `CredentialsContainer` class, but I'm not sure it's worth it, and I didn't want to bloat the scope here.
I will make a task.

Reply:
I think this can be followed-up with the same task as below: https://app.asana.com/0/1204556816597738/1207323488899175/f